products/sce/autoscaling/enable-autoscaling.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: 'Set Up Autoscaling'
+---
+
+You can enable the autoscaling feature in SaladCloud through the portal or via the API. Here's how:
+
+## Using the Portal
+
+1. **Create or Modify a Container Group**:
+   - In the SaladCloud portal, create a new container group or edit an existing one.
+2. **Enable the Salad Queue**:
```

Comment:
I think we should add a comment here that shares that you can only configure your Job Queue connection with the "Create Container" function. If the user doesn't connect to a job queue on create, they will not be able to retrospectively add later in the "edit container" wizard. This is different from the autoscaling settings, which can be added or edited later on any container group with an active queue connection.